src/app/view-project/view-project.component.html

```diff
@@ -0,0 +1,51 @@
+<h1> Your Projects </h1>
+<table class="table">
+  <thead>
+  <tr>
+    <th scope="col">#</th>
+    <th scope="col">Project Options</th>
+    <th scope="col">Project Name</th>
+    <th scope="col">API base url</th>
+    <th scope="col">Description</th>
+    <th scope="col">Resources</th>
+
+
+  </tr>
+  </thead>
+  <tbody>
+  <ng-container *ngFor="let project of projects | async; index as i">
+    <tr>
+      <th scope="row">{{i + 1}}</th>
+      <td>
+        <svg (click)="deleteProject(project.name)" xmlns="http://www.w3.org/2000/svg" width="16" height="16"
```

Comment:
Do you have a reason why do you need the svg embedded inside your component? if not, use font awesome or an image to display this.